src/main/java/net/minecraftforge/resource/VanillaResourceType.java

```diff
@@ -51,4 +51,9 @@
      * Used when all shaders are reloaded.
      */
     SHADERS,
+    
+    /**
+     * Used when recipes are reloaded.
+     */
+    RECIPES,
```

Comment:
Is this used?

Reply:
I was going to do something with it, then realized the selectivity was only on the client.  It may want to be expanded now that resources are not a client-only thing.